src/main/resources/crafter/studio/upgrade/4.1.x/4.1.2/system/global-permission-mappings-config.xslt

```diff
@@ -0,0 +1,111 @@
+  ~ it under the terms of the GNU General Public License version 3 as published by
+  ~ the Free Software Foundation.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="2.0">
+
+    <!-- to keep the right formatting -->
+    <xsl:output method="xml" indent="yes" />
+    <xsl:strip-space elements="*"/>
+
+    <!-- copy all elements -->
+    <xsl:template match="node() | @*">
+        <!-- insert line breaks before comments -->
+        <xsl:if test="self::comment()">
+            <xsl:text>&#10;</xsl:text>
+        </xsl:if>
+        <xsl:copy>
+            <xsl:apply-templates select="node() | @*"/>
+        </xsl:copy>
+        <!-- insert line breaks after comments -->
+        <xsl:if test="self::comment()">
+            <xsl:text>&#10;</xsl:text>
+        </xsl:if>
+    </xsl:template>
+
+    <xsl:template match="permissions/role[@name='system_admin']/rule/allowed-permissions/permission[. = 'create-site']">
+        <xsl:element name="permission"><xsl:text>create_site</xsl:text></xsl:element><xsl:text>&#10;</xsl:text>
+    </xsl:template>
+    <xsl:template match="permissions/role[@name='system_admin']/rule/allowed-permissions/permission[. = 'site_delete']">
+        <xsl:element name="permission"><xsl:text>delete_site</xsl:text></xsl:element><xsl:text>&#10;</xsl:text>
+    </xsl:template>
+    <xsl:template match="permissions/role[@name='system_admin']/rule/allowed-permissions">
+        <xsl:copy>
+            <xsl:copy-of select="@*"/>
+            <xsl:apply-templates/>
+            <xsl:if test="not(permission = 'content_create')">
+                <xsl:element name="permission">
+                    <xsl:text>content_create</xsl:text>
+                </xsl:element>
+                <xsl:text>&#10;</xsl:text>
+            </xsl:if>
+            <xsl:if test="not(permission = 'get_publishing_queue')">
+                <xsl:element name="permission">
+                    <xsl:text>get_publishing_queue</xsl:text>
+                </xsl:element>
+                <xsl:text>&#10;</xsl:text>
+            </xsl:if>
+            <xsl:if test="not(permission = 'cancel_publish')">
+                <xsl:element name="permission">
+                    <xsl:text>cancel_publish</xsl:text>
+                </xsl:element>
+                <xsl:text>&#10;</xsl:text>
+            </xsl:if>
+            <xsl:if test="not(permission = 'Change Content Type')">
+                <xsl:element name="permission">
+                    <xsl:text>Change Content Type</xsl:text>
```

Comment:
Please switch this to `change_content_type` like you did `create-site`

Reply:
Updated. I also change the following permissions to match our convention:
```
S3 Read => s3_read
S3 Write => s3_write
```